src/django_components/component_media.py

```diff
@@ -399,11 +532,6 @@ def get_context_data(self):
         comp_cls, comp_media, inlined_attr="css", file_attr="css_file", comp_dirs=comp_dirs, type="static"
     )
 
-    media_cls = comp_media.media_class or MediaCls
-    media_js = getattr(comp_media.Media, "js", [])
-    media_css = getattr(comp_media.Media, "css", {})
-    comp_media.media = media_cls(js=media_js, css=media_css) if comp_media.Media is not None else None
```

Comment:
Previously, the `media` instance contained only the JS / CSS files from the current Component class, so it was enough to just instantiate it. Now, the [`_get_comp_cls_media`](https://github.com/EmilStenstrom/django-components/pull/890/files#diff-bdb1917504a9c50a51e7fdf49498d5db3ce0233be9eb81b13938d4b3dcd5efc3R385) travels up the Component's parent classes to find all the JS / CSS. And so the child component class end up inheriting them all.